api/Navigator.json

```
@@ -3039,16 +3030,1216 @@
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": "62"
+                "version_added": "2"
               },
               "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "15"
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "scheme_parameter_ftp": {
+          "__compat": {
+            "description": "<code>scheme</code> parameter supports <code>ftp</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
                 "version_added": false
               },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "98"
+              },
+              "firefox_android": {
+                "version_added": "98"
+              },
               "ie": {
                 "version_added": false
               },
               "opera": {
-                "version_added": "67"
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```

The single implementations are going to be experimental. Would you mind double checking each of these?

```suggestion
              "experimental": true,
```

---

Cheers. You got all except one. Fixed.